### PR TITLE
added Zapper payment icon

### PR DESCRIPTION
**Pull request by MasterOdious**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/zapper.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-zapper">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/zapper.svg]: Is it possible to add more space around the icon? It looks a bit tight.

[alt](https://screenshot.click/27-45-j5ax8-w7vzk.png)

Comment (tauthomas01) [reply app/assets/images/payment_icons/zapper.svg]: Also, can you run your SVG through SVGO? The instructions are in the [Contribution guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

Comment (MasterOdious) [reply app/assets/images/payment_icons/zapper.svg]: used our shorter logo, added spacing, ran svgo

File: app/assets/images/payment_icons/zapper.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-zapper"><title id="pi-zapper">Zapper</title><style type="text/css"/><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M17.6 16.8h8.5c1.3 0 2.4 1.1 2.4 2.3 0 1.3-1.1 2.4-2.4 2.4H11.9c-.6 0-1.2-.2-1.7-.7-.9-.9-.9-2.4 0-3.3L24.5 3.3c.9-.9 2.4-.9 3.3 0 .9.9.9 2.4 0 3.3L17.6 16.8zm-.9-9.7h-4.8c-1.3 0-2.4-1.1-2.4-2.3 0-1.3 1.1-2.3 2.4-2.3h9.7l-4.9 4.6z" fill-rule="evenodd" clip-rule="evenodd" fill="#005cb9"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/zapper.svg]: Can we remove `<style type="text/css"/>`? It's not used anywhere.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-zapper"><title id="pi-zapper">Zapper</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M17.6 16.8h8.5c1.3 0 2.4 1.1 2.4 2.3 0 1.3-1.1 2.4-2.4 2.4H11.9c-.6 0-1.2-.2-1.7-.7-.9-.9-.9-2.4 0-3.3L24.5 3.3c.9-.9 2.4-.9 3.3 0 .9.9.9 2.4 0 3.3L17.6 16.8zm-.9-9.7h-4.8c-1.3 0-2.4-1.1-2.4-2.3 0-1.3 1.1-2.3 2.4-2.3h9.7l-4.9 4.6z" fill-rule="evenodd" clip-rule="evenodd" fill="#005cb9"/></svg>
```

Comment (MasterOdious) [reply app/assets/images/payment_icons/zapper.svg]: Thank you. made the change

**Pull request merged**